features/fixtures/formatters/failed-rule.json.ts

```diff
@@ -0,0 +1,39 @@
+module.exports = [
```

Comment:
Do we really need these? I only created json fixtures to make it easy to see if we changed something for backward compatibility. Maybe just one for a rule would be enough?

Reply:
Removed so we are now only adding one json fixture for rule happy path

Reply:
Still seeing failed rule and rejected pickle rule

Reply:
Ah okay - I think I thought you were only talking about the fixtures for the JSON formatter. Should I scale back the ones for the message formatter as well?

Reply:
Haha understandable. I meant the fixtures in general (both json + message formatter). Think just a passed version of json / messages is sufficient

Reply:
No worries, done that now.